mon-pix/app/components/routes/campaigns/fill-in-participant-external-id.hbs

```diff
@@ -0,0 +1,34 @@
+<main class="rounded-panel rounded-panel--strong rounded-panel--over-background-banner fill-in-participant-external-id__container">
+  <h1 class="fill-in-participant-external-id-text fill-in-participant-external-id-text__title">{{t 'pages.fill-in-participant-external-id.first-title'}}</h1>
+  <p class="fill-in-participant-external-id-text fill-in-participant-external-id-text__announcement">
+    {{t 'pages.fill-in-participant-external-id.announcement'}}
+  </p>
+
+  <form {{on 'submit' this.submit}} class="fill-in-participant-external-id__form">
+    <div class="fill-in-participant-external-id-form__section">
+      <div class="fill-in-participant-external-id__form-field">
+        <label for="id-pix-label">{{@campaign.idPixLabel}}</label>
+        <Input required @id="id-pix-label" @value={{this.participantExternalId}} />
+      </div>
+
+      <button type="submit" disabled={{this.isLoading}} class="button button--big">
+        {{#if this.isLoading}}
+          <span class="loader-in-button">&nbsp;</span>
```

Comment:
&nbsp revient 24 fois dans l'app dont 20 fois pour le "loader-in-button". 
Ne peut-on pas remplacer ce caractère spéciale par une propriété CSS (margin ou padding) pour créer un "espace-forcé" ?

Reply:
ça marche pô.